specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-04-01/registries.json

```diff
@@ -710,6 +721,34 @@
       },
       "additionalProperties": false
     },
+    "ManagedResourceGroupSettings": {
+      "description": "Managed resource group settings",
+      "type": "object",
+      "properties": {
+        "assignedIdentities": {
+          "description": "List of assigned identities for the managed resource group",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedResourceGroupAssignedIdentities"
+          },
+          "x-nullable": true,
+          "x-ms-identifiers": []
+        }
+      },
+      "additionalProperties": false
+    },
+    "ManagedResourceGroupAssignedIdentities": {
```

Comment:
Would it make sense to reuse a definition from commontypes v5's managedidentity.json for this, instead of defining a new one?

Reply:
I tried to use a definition in v5's managedidentity.json, that will bring in a break change of our 2024-01-01-preview version.